### DISCUSSION TITLE
Recommendations for deploying packages across grid

### DISCUSSION THREAD

**Discussion by darth-drew** [Security-Onion-Solutions/securityonion#3351] [Unsupported Versions] [answered]
I am working on deploying an additional, non-standard package across an airgapped grid. I assumed the best way to do this was to point the manager to a yum server on our network, but it appears that salt is overwriting the custom airgap_repo.repo that I wrote.

So for now I manually pushed the .rpm I am interested in getting on all the forward nodes to the manager (/nsm/repo) but a simple yum install myPackage on the sensors also does not work.

What are the recommended best practices for deploying a new package across an airgrap grid? I know the manager is supposed to have some role, but it seems to me this is only packages distributed by the ISO. How would I customize the airgap_repo file to persistently include an external server without breaking anything? Any guidance is appreciated. Thanks again.

**Comment by TOoSmOotH** [comment, marked as answer]
Drop the RPMs into /nsm/repo and then run:

`sudo createrepo /nsm/repo`

**Reply by darth-drew** [reply]
That couldn't have been easier. Thanks!
